templates/miq-template.yaml

```diff
@@ -176,6 +241,10 @@ objects:
            name: "${NAME}-region"
            persistentVolumeClaim:
              claimName: ${NAME}-region
+         -
+           name: "${NAME}-assets"
```

Comment:
This PV is shared then? Is that because we do the compilation on this container and then use the assets on the apache one?

If so, why not do the compilation on the apache container?